src/Raven.Client/Documents/Session/Tokens/TimeSeriesIncludesToken.cs

```diff
@@ -43,12 +44,12 @@ public override void WriteTo(StringBuilder writer)
                 .Append(", ");
 
             writer.Append("'")
-                .Append(_range.From.GetDefaultRavenFormat())
+                .Append((_range.From ?? DateTime.MinValue).GetDefaultRavenFormat())
```

Comment:
I think we need to support includes like `timeseries(<range_name>)`. Requiring here a date will be a problem for non-clr clients